en-US/firefox-ios.xliff

```diff
@@ -1874,6 +1953,36 @@
         <target>Clear Private Data</target>
         <note>Title displayed in header of the setting panel.</note>
       </trans-unit>
+      <trans-unit id="Settings.DNT.OptionAlwaysOn">
+        <source>Always</source>
+        <target>Always</target>
+        <note>DNT Settings option for always on</note>
+      </trans-unit>
+      <trans-unit id="Settings.DNT.OptionOnWithTP">
+        <source>Only when using Tracking Protection</source>
+        <target>Only when using Tracking Protection</target>
+        <note>DNT Settings option for only turning on when Tracking Protection is also on</note>
+      </trans-unit>
+      <trans-unit id="Settings.DNT.Title">
+        <source>Send websites a “Do Not Track” signal that you don’t want to be tracked</source>
```

Comment:
You're mixing up proper apostrophes and straight quotes in strings. This string uses ’, for example intro.Slides.Private.Title uses '.